src/tab/Tab.tsx

```diff
@@ -49,6 +49,13 @@ function Tab({
   const [activeTabIndex, setActiveTabIndex] = useState(initialActiveTabIndex);
   const tabClassName = classNames("tab", customClassName);
 
+  let activeTabIndexToUse = activeTabIndexFromProps === undefined
+    ? activeTabIndex
+    : activeTabIndexFromProps;
```

Comment:
Would that work?

```suggestion
  let activeTabIndexToUse = activeTabIndexFromProps || activeTabIndex
```

Reply:
Yes, this works 👍

Reply:
What if `activeTabIndexFromProps` is 0? Does it still work as expected?